package.json

```diff
@@ -40,21 +40,25 @@
     "pg_permissions": "exec ./utils/postgraphile_launch_permissions.sh",
     "dev_pg": "concurrently --kill-others-on-fail \"yarn dev\"  \"yarn pg\"",
     "dev_pg_permissions": "concurrently --kill-others-on-fail \"yarn dev\"  \"yarn pg_permissions\"",
-    "database_init": "./database/initialise_database.sh && ./database/insert_data.sh",
-    "postdatabase_init": "./database/post_data_insert.sh",
+    "database_init": "./database/initialise_database.sh tmf_app_manager && ./database/insert_data.sh",
+    "postdatabase_init": "./database/turn_on_row_level_security.sh && ./database/post_data_insert.sh && rm -rf ./files && mkdir ./files",
     "build_plugins": "node ./utils/pluginScripts/buildPlugins.js",
     "push_docs": "exec ./utils/push_docs_to_wiki.sh",
     "gui": "cd ./src/modules/expression-evaluator && yarn gui",
     "generate": "graphql-codegen --config codegen.yml",
     "postinstall": "node ./utils/pluginScripts/yarnInstallPlugins.js",
-    "take_snapshot": "node ./database/takeSnapshot.js",
-    "use_snapshot": "node ./database/useSnapshot.js"
+    "snapshot": "ts-node ./database/snapshotCLI.ts"
   },
   "dependencies": {
     "@graphile-contrib/pg-simplify-inflector": "^6.0.0",
     "@openmsupply/deep-comparison": "^1.0.1",
+    "@types/adm-zip": "^0.4.34",
     "@openmsupply/expression-evaluator": "^1.8.0",
     "@types/jsonwebtoken": "^8.5.0",
+    "@types/lodash": "^4.14.169",
```

Comment:
Add lodash to imports?

Reply:
cb1eec9d61fefc6686e4f9e6adcefb234f14c9ac